src/consensus.rs

```diff
@@ -229,8 +282,30 @@ pub struct Consensus {
 }
 
 impl Consensus {
-    // Reset bft_round_state for the next round of consensus.
-    fn finish_round(&mut self, commit: Option<QuorumCertificate>) -> Result<(), Error> {
+    fn next_leader(&self) -> Result<ValidatorPublicKey> {
```

Comment:
```suggestion
    fn next_leader(&self) -> Result<NextLeader> {
```
ou c'est trop verbeux? comme y'a le type en haut j'aurais tendance à etre explicite